scripts/functions.py

```diff
@@ -106,4 +107,404 @@ def daylightSavings(inData):
+        for tech in techs:
+            techName = 'MIN' + tech + 'INT'
+            outList.append(techName)
+
+    # Return list of min Technologes
+    return outList
+
+def getRNWtechs(regions, techs):
+    # PURPOSE: Creates all the RNW naming technologies
+    # INPUT:   regions =  Dictionary holding region as the key and subregion as the values in a list
+    #          techs =    List of the technologies to print over 
+    # OUTPUT:  outList =  List of all the RNW technologies
+
+    # list to hold technologies
+    outList = []
+
+    # Loop to create all technology names
+    for region, subregions in regions.items():
+        for subregion in subregions:
+            for tech in techs:
+                techName = 'RNW' + tech + region + subregion
+                outList.append(techName)
+    
+    # Return list of rnw Technologes
+    return outList
+
+def getTRNtechs(csvPath):
+    # PURPOSE: Creates all the TRN naming technologies
+    # INPUT:   csvPath = The location of the datafile, as a path
+    # OUTPUT:  outList =  List of all the TRN technologies
+
+    # list to hold technologies
+    outList = []
+
+    #Read in the trade csv datafile 
+    df = pd.read_csv(csvPath)
+    outList = df['TECHNOLOGY'].tolist()
+    outList = list(set(outList)) # remove duplicates
+    
+    # Return list of rnw Technologes
+    return outList
+
+def getRNWfuels(regions, techs):
+    # PURPOSE: Creates fuels names for Renewable technologies
+    # INPUT:   regions =  Dictionary holding region as the key and subregion as the values in a list
+    #          techs =    List of the technologies to print over 
+    # OUTPUT:  outList =  List of all the RNW technologies
+
+    # list to hold technologies
+    outList = []
+
+    # Loop to create all technology names
+    for region, subregions in regions.items():
+        for subregion in subregions:
+            for tech in techs:
+                fuelName = tech + region + subregion
+                outList.append(fuelName)
+    
+    # Return list of rnw fuels
+    return outList
+
+def getMINfuels(regions, techs, generateInternational):
+    # PURPOSE: Creates Fuel names for Min technologies
+    # INPUT:   regions =  Dictionary holding region as the key and subregion as the values in a list
+    #          techs =    List of the technologies to print over 
+    #          generateInternational = True/False for whether function needs to create all
+    #                                  technology names for international import/export
+    # OUTPUT:  outList =  List of all the MIN Fuels 
+
+    # list to hold technologies
+    outList = []
+
+    # Loop to create all technology names based on region
+    for region in regions:
+        for tech in techs:
+            fuelName = tech + region
+            outList.append(fuelName)
+    
+    # Created once in Canada scripts
+    # Loop to create all technology names for international import/export
+    if generateInternational:

```

Comment:
If it were its own function, the generateInternational boolean would still be necessary, leaving five lines, which is a bit lean for a function used in only one place (as far as I can see). That being said, maybe this will be affected by #70? Not completely sure.